addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -545,6 +545,12 @@
             t-att-data-can-publish="channel.can_publish">
             <i class="fa fa-cloud-upload mr-1"/>Upload new content
         </a>
+        <a class="btn btn-secondary py-1 o_wslides_js_slide_section_add"
+            title="Add Section" role="button"
+            aria-label="Add Section" href="#"
+            t-attf-channel_id="#{channel.id}">
```

Comment:
You can just do `t-att-channel_id="channel.id"`

(I know you copied from the other side).